packages/api/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export * as routes from "./routes";
 export * from "./interface";
 export {getClient, HttpClient, HttpClientOptions, HttpError} from "./client";
+export {getClient as getKeymanagerClient} from "./keymanager/client";
```

Comment:
Don't import in the root index. The above `keymanager.js` file in the root are meant to allow to only import this subset of the API